docs/behringer_mic2usb/README.md

```diff
@@ -0,0 +1,81 @@
+# Behringer MIC2 USB
+
+Very nice sound qulity with combination of Digitech BP355 bass processor.  
+
+Please double-check your ini-files settings before reporting an issue.  
+
+## Config files
+
+**RS_ASIO.ini**
+
+```ini
+[Config]
+EnableWasapiOutputs=0
+EnableWasapiInputs=0
+EnableAsio=1
+
+[Asio]
+; available buffer size modes:
+;    driver - respect buffer size setting set in the driver
+;    host   - use a buffer size as close as possible as that requested by the host application
+;    custom - use the buffer size specified in CustomBufferSize field
+BufferSizeMode=custom
+CustomBufferSize=192
+[Asio.Output]
+Driver=ASIO4ALL v2
+BaseChannel=0
+EnableSoftwareEndpointVolumeControl=1
+EnableSoftwareMasterVolumeControl=1
+SoftwareMasterVolumePercent=100
+
+[Asio.Input.0]
+Driver=ASIO4ALL v2
+Channel=0
+EnableSoftwareEndpointVolumeControl=1
+EnableSoftwareMasterVolumeControl=1
+SoftwareMasterVolumePercent=100
+
+[Asio.Input.1]
+Driver=
+Channel=1
+EnableSoftwareEndpointVolumeControl=1
+EnableSoftwareMasterVolumeControl=1
+SoftwareMasterVolumePercent=100
+```
+
+**Rocksmith.ini**
+
+```ini
+[Audio]
+EnableMicrophone=1
+ExclusiveMode=1
+LatencyBuffer=1
+ForceDefaultPlaybackDevice=Stereo Headset 1
```

Comment:
Can you please change this to `ForceDefaultPlaybackDevice=` to avoid people copying your personal settings?
We don't want issues opening because of this, even if this probably doesn't make a difference without Wasapi :)